examples/ipv6-eks-cluster/main.tf

```diff
@@ -70,14 +71,21 @@ locals {
 
 module "aws_vpc" {
   source  = "terraform-aws-modules/vpc/aws"
-  version = "v3.2.0"
+  version = "v3.13.0"
```

Comment:
Probably irrelevant to this PR, but should we consider updating the AWS_VPC version for other examples as-well?

Reply:
I had to upgrade the VPC module version for IPv6 support. We can take a decision on updating other examples once we decide with our new example approach whether to use VPC or not